### PR TITLE
pnmhisteq, pnmhistmap, pnmindex, pnminvert: add pages

### PR DESCRIPTION
**Pull request by gutjuri**

### CODE REVIEW

File: pages/common/pnmindex.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+> Build a visual index of multiple PNM images.
+> See also: `pamundice`.
+> More information: <https://netpbm.sourceforge.net/doc/pnmindex.html>.
+
+- Produce an image containing thumbnails of the specified images in a grid:
+
+`pnmindex {{path/to/input1.pnm path/to/input2.pnm ...}} > {{path/to/output.pnm}}`
+
+- Specify the size of the (quadratic) thumbnails:
```
Comment (kbdharun) [inline pages/common/pnmindex.md:11]:
```suggestion
- Specify the size of the quadratic thumbnails:
```
Do we need to enclose it inside brackets?

File: pages/common/pnmindex.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+`pnmindex -size {{50} {{path/to/input1.pnm path/to/input2.pnm ...}} > {{path/to/output.pnm}}`
```
Comment (kbdharun) [inline pages/common/pnmindex.md:13]:
```suggestion
`pnmindex -size {{50}} {{path/to/input1.pnm path/to/input2.pnm ...}} > {{path/to/output.pnm}}`
```

**Review by kbdharun — APPROVED** [review]
Thanks for your contribution. I have some suggestions for this page.

File: pages/common/pnmindex.md (hunk 1 above)
Comment (gutjuri) [reply pages/common/pnmindex.md:11]: I wanted to express that this does not only set the sizes of the quadratic thumbnail, but that all thumbnails are quadratic.

**Review by blueskyson — APPROVED** [review]

**Pull request merged**